src/main/java/seedu/address/model/procedure/Date.java

```diff
@@ -35,7 +34,7 @@
     public Date(String date) {
         requireNonNull(date);
         checkArgument(isValidDate(date), MESSAGE_CONSTRAINTS);
-        validDate = LocalDate.parse(date, DateTimeFormatter.ofPattern("dd/MM/uuuu")); // need to check
+        validDate = LocalDateTime.parse(date, DateTimeFormatter.ofPattern("dd/MM/uuuu HH:mm"));
```

Comment:
Let's make this formatter pattern to be a final static variable so we can re-use it multiple times! We can make it public or private. For now, private should be good enough because no other classes are using it, but we might use it in the future for loading.